src/components/overlay/readme.md

```diff
@@ -0,0 +1,113 @@
+The `sbb-overlay` component provides a way to present content on top of the app's content and it's similar to the [sbb-dialog](/docs/components-sbb-dialog--docs).
+It offers the following features:
+
+- disables scrolling of the page content while open;
+- manages focus properly by setting it on the first focusable element;
+- has a close button, which is always visible;
+- can display a back button;
+- adds the appropriate ARIA roles automatically.
+
+```html
+<sbb-overlay id="my-overlay">
+  <p>Overlay content.</p>
+</sbb-overlay>
+```
+
+## Slots
+
+There is only one unnamed slot to provide the overlay content.
+
+## Style
+
+Setting the `expanded` property will cause the `sbb-overlay` component to take up the full width of the page.
+
+It's possible to display the component in `negative` variant using the self-named property.
+
+```html
+<sbb-overlay negative>
+  <p>Overlay content.</p>
+</sbb-overlay>
+```
+
+## Interactions
+
+In order to show the overlay, you need to call the `open(event?: PointerEvent)` method on the `sbb-overlay` component.
+It is necessary to pass the event object to the `open()` method to allow the overlay to detect
+whether it has been opened by click or keyboard, so that the focus can be better handled.
+
+```html
+<sbb-button
```

Comment:
I think we should once more consider also supporting passing a trigger as property / id? (maybe also as a following story)

Reply:
I'll create a card for this 👍